Makefile

```diff
@@ -26,7 +26,12 @@ mlmd-proto:
 	mv ml_metadata/proto/*.go ml_metadata/
 
 metadata-docker-image:
-	docker build -t gcr.io/kubeflow-images-public/metadata .
+	@if test $(shell uname -m) = "aarch64"; then \
```

Comment:
Can you add a new build target like `docker-image-aarch64`? The build target doesn't have to tie to build system info.

Reply:
Yes for sure, and I have updated the commit :)